spec.bs

```diff
@@ -365,7 +387,10 @@ A [=component=] has an associated <dfn for=component>group name list</dfn>, a [=
   1. If |input| is null, then set |input| to "`*`".
   1. Let |part list| be the result of running [=parse a pattern string=] given |input|, |options|, and |encoding callback|.
   1. Let (|regular expression string|, |name list|) be the result of running [=generate a regular expression and name list=] given |part list| and |options|.
-  1. Let |regular expression| be [$RegExpCreate$](|regular expression string|, "`u`").  If this throws an exception, catch it, and throw a {{TypeError}}.
+  1. Let |flags| be an empty string.
+  1. If |options|["ignoreCase"] is true then set |flags| to "`ui`".
```

Comment:
Since `|options|` is an internal struct the prefered syntax is a bit different than the webidl object.  Instead write it as `|options|'s [=options/ignore case=]`.